macos-manual.md

```diff
@@ -0,0 +1,112 @@
+### OSX
+
+#### 1. Install dependencies
+
+###### a) Build tools
+
+    install xcode
+    Qt5 https://www.qt.io/download-open-source/ TODO see back
+    TODO git
```

Comment:
This file contains instructions which were in main README.md before, I have not changed instructions for manual install. I agree the instructions are not optimal, I shall remove unnecessary steps.